### PR TITLE
Rename monitor to s4s and add first customer

### PR DESCRIPTION
**Pull request by mattgauntseo-sentry**
This switches the name of monitor to s4s (just easier to understand for sentaurs).

The `customer-1` pipeline is the first customer pipeline being added to relay, this should follow the same process as s4s.

I've also removed the rollback pipeline as it doesn't work how relay team would like it to and in its current form it doesn't work with manual steps for pops.

#skip-changelog

### CODE REVIEW

**Comment by mattgauntseo-sentry** [comment]
NOTE: The error in the validate deployment pipelines is a known issue and I need to investigate it.

**Review by olksdr — APPROVED** [review]

File: gocd/generated-pipelines/rollback-relay.yaml
Comment (jjbayer) [inline gocd/generated-pipelines/rollback-relay.yaml:1]: Why is this deleted?

File: gocd/templates/jsonnetfile.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
           "subdir": "libs"
         }
       },
-      "version": "v1.1.9"
+      "version": "v1.2.2"
```
Comment (jjbayer) [inline gocd/templates/jsonnetfile.json:11]: Does this version version bump cause `relay-customer-1.yaml` to be generated?

**Review by jjbayer — COMMENTED** [review]
Would this be an independent pipeline, or would it be automatically triggered on every Relay deploy like S4S?

File: gocd/templates/jsonnetfile.json (hunk 1 above)
Comment (mattgauntseo-sentry) [reply gocd/templates/jsonnetfile.json:11]: yes it does

File: gocd/generated-pipelines/rollback-relay.yaml
Comment (mattgauntseo-sentry) [reply gocd/generated-pipelines/rollback-relay.yaml:1]: This is because the rollback pipeline doesn't work with the relay pipedream because of the manual promotion to pops step.

The built in pipeline works by re-running the primary deployment stage in each region, however, if you have a manual approval step, that doesn't work because it'll deploy-relay but GoCD will not offer the promote-to-pops step.

Further - when asking about this in the slack channel, it seemed like a rollback would first deploy to pops and then deploy relay (i.e. operate in reverse)
